tools/AutomationTestFramework/ManageScripts/GenerateAndUploadTestHelperModule.ps1

```diff
@@ -1,47 +0,0 @@
-# ----------------------------------------------------------------------------------
```

Comment:
looks like you need to pull the latest from preview

Reply:
Pulled.